src/Artsy/Router/Components/PreloadLink.tsx

```diff
@@ -104,11 +104,7 @@ const _PreloadLink: React.SFC<PreloadLinkProps> = preloadLinkProps => {
      *   {
      *     path: '/home',
      *     Component: () => <div>Home!</div>
-     *     query: graphql`
-     *       query routes_HomeQuery {
-     *         ...
-     *       }
-     *     `
+     *     query: ...
```

Comment:
The `vscode-apollo` extension doesn’t properly parse source, instead it uses regexps… so it’s also picking up this comment and throwing a syntax error. Not going to be bothered by this, for now.